ui/views/sync.jsx

```diff
@@ -131,32 +134,41 @@ function createRenderer (graph, el) {
       nodeClass.push('is-user')
     }
     if (isFriend) nodeClass.push('is-friend')
-    if (node.data.isConnected) nodeClass.push('is-connected')
+    if (isConnectedPeer) nodeClass.push('is-connected')
     if (isRelayPeer) radius = 10
 
-    return ngraphSvg.svg("circle", {
+    const circle = ngraphSvg.svg("circle", {
       r: radius,
       cx: 2*radius,
       cy: 2*radius,
       class: nodeClass.join(' '),
-      peerId: node.data.id,
+      id: node.data.id,
       text: node.data ? node.data.name : undefined
     })
+    const link = ngraphSvg.svg("a")
+    //const link = ngraphSvg.svg("a", { link: '#/profile/'+encodeURIComponent(node.data.id) })
```

Comment:
here's the line that adds a non-react router link and throws an issue when you click on links 
says: 

```
TypeError: Cannot set property 'innerHTML' of null
    at SVGSVGElement.<anonymous> (/home/mix/projects/patchwork/ui/views/sync.jsx:86:55)
```